solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -290,13 +290,25 @@ library Relay {
         view
         returns (bool)
     {
-        uint256 relayEntryTimeout = (dkgGroupSize *
+        uint256 _relayEntryTimeout = (dkgGroupSize *
             self.relayEntrySubmissionEligibilityDelay) +
             self.relayEntryHardTimeout;
 
         return
             isRequestInProgress(self) &&
-            block.number > self.currentRequest.startBlock + relayEntryTimeout;
+            block.number > self.currentRequest.startBlock + _relayEntryTimeout;
+    }
+
+    /// @notice Calculates a relay entry timeout.
+    /// @return Relay entry timeout in blocks.
+    function relayEntryTimeout(Data storage self)
+        internal
+        view
+        returns (uint256)
+    {
+        return
+            (dkgGroupSize * self.relayEntrySubmissionEligibilityDelay) +
+            self.relayEntryHardTimeout;
```

Comment:
If we get rid of a stale group concept, we no longer need this function, right?

Reply:
Yup, removed 921f30b